src/map/lua/lua_baseentity.cpp

```diff
@@ -6765,9 +6764,8 @@ void CLuaBaseEntity::addQuest(uint8 questLogID, uint16 questID)
     if (questLogID < MAX_QUESTAREA && questID < MAX_QUESTID)
     {
         uint8 current  = PChar->m_questLog[questLogID].current[questID / 8] & (1 << (questID % 8));
```

Comment:
CI complains of the alignment here since the line it was aligned to below it no longer exists